packages/web3torrent/src/library/paid-streaming-extension.ts

```diff
@@ -96,7 +96,17 @@ export abstract class PaidStreamingExtension implements Extension {
   }
 
   protected executeExtensionCommand(command: PaidStreamingExtensionNotices, data = {}) {
-    this.wire.extended(this.name, bencode.encode({msg_type: 0, command, data}));
+    if (!this.peerAccount) {
+      log(
+        'WARNING: Peer does not support Web3Torrent - This client will block all non-web3torrent leechers.'
+      );
+      this.messageBus.emit(
+        PaidStreamingExtensionEvents.WARNING,
+        new Error('!>Peer does not support Web3Torrent')
```

Comment:
Can we get rid of the `!>` prefixes?

Reply:
It follows the _format_ so to speak that we had before. It signals something that's on the peer, and not in _this client_.
I would remove it, but in another PR so that we can define a _format_ so to speak.

Reply:
We could have a `PeerError` typed error for this.

Reply:
Sure!